build.assets/Dockerfile

```diff
@@ -96,7 +94,7 @@ RUN mkdir -p /opt && cd /opt &&  \
 # 4. Fast, language-dependent dependencies
 # 5. Multi-stage layer copies
 
-FROM ubuntu:18.04 AS buildbox
+FROM ubuntu:20.04 AS buildbox
```

Comment:
I might have missed it in the original discussion, but wouldn't bumping version to 20.04 raise our minimum glibc version requirement?

Reply:
I want to backport this as Ubuntu 18.04 is EOL. Bumping the glibc version on the buildbox won't change the glibc version required by Teleport as we're using CentOS 7 for Teleport and the new Connect image for Teleport Connect releases.